src/utils/index.js

```diff
@@ -35,3 +35,31 @@ export function getPipelineRunStatusIcon({ reason, status }) {
 
   return icon ? <Icon name={icon} className="status-icon" /> : null;
 }
+
+export function taskRunStep(selectedStepId, taskRun) {
+  if (!taskRun || !taskRun.steps) {
+    return {};
+  }
+  const step = taskRun.steps.find(s => s.id === selectedStepId);
+  if (!step) {
+    return {};
+  }
+
+  const { id, stepName, stepStatus, status, reason, ...definition } = step;
+
+  return {
+    definition,
+    reason,
+    stepName,
+    stepStatus,
```

Comment:
What's the difference between `status` and `stepStatus` here?

Reply:
Step status is actually the value for the step in the task run (
```
    {
      terminated: {
        exitCode: 1,
        'Error',
        startedAt: '2019',
        finishedAt: '2019',
        containerID: 'containerd://testid'
      }
    }
```
for example

status is the actual succeeded/failed/waiting ...

Bad naming for sure but preexisting. Changing it in this PR might mean hitting other components. Mind if I address it separately ?

Reply:
ah yes, ok let's follow up before this gets forgotten 👍